### ISSUE TITLE
Alpine severity unknown

### ISSUE THREAD

**Issue by c4tz** [quay/clair#850]
### Description of Problem / Feature Request

For alpine based images, all CVEs have the severity "unkown". This wasn't the case before (our last report from 05/04/2019 show specific severities).

### How to reproduce

```shell_session
$ mkdir $PWD/clair_config
$ curl -L https://raw.githubusercontent.com/coreos/clair/master/config.yaml.sample -o $PWD/clair_config/config.yaml
$ docker run -d -e POSTGRES_PASSWORD="" -p 5432:5432 postgres:9.6
$ docker run --net=host -d -p 6060-6061:6060-6061 -v $PWD/clair_config:/config quay.io/coreos/clair:latest -config=/config/config.yaml

(wait for db initialised/seeded)

$ curl "http://127.0.0.1:6060/v1/namespaces/alpine:v3.5/vulnerabilities?limit=200"
```
-> All entries have the attribute `Severity: "Unknown"`. 

Maybe the Severity could be parsed from `nvd`, only if it has the status "Unknown"?

### Environment

- Clair version/image: 2.0.8
- Clair client name/version: direct API call
- Host OS: Docker on Arch Linux
- Kernel (e.g. `uname -a`): 5.3.1-arch1-1-ARCH

**Comment by ldelossa** [comment]
@c4tz fixed in v2.1.0 and latest master

**Comment by degilq** [comment]
hi all we have same problem with version quay.io/coreos/clair:v2.1.1

alpine all results are [Unknown] severity
